LondonBritishLibrary/orient/BLorient546.xml

```diff
@@ -31,21 +31,165 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 133</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
+
+<msItem xml:id="ms_i1">
+<locus from="3r"/>
+<title type="complete" xml:lang="gez" ref="LIT1960Mashaf">The Qǝddāse or Missal</title>
+<textLang mainLang="gez"></textLang>
+<notatedMusic><desc>Extensive musical notation.</desc></notatedMusic>
+</msItem>
+
+<msItem xml:id="ms_i1.1">
+<locus from="3r"/>
+<title type="complete" xml:lang="gez" ref="LIT1716Kidanz">Daily prayer, etc.</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="26r"/>
+<title type="complete" xml:lang="gez" ref="LIT1960Mashaf#SerataQeddase">The ordinary Canon of the Mass</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="52v"/>
+<title type="complete" xml:lang="gez" ref="LIT3189Anapho">The anaphoras</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
```

Comment:
The closing tag should be moved after the last msItem nested inside. The numbering is the second thing, but if one of the msItems has subitems, those should be nested inside the main item.